source/style-guide/style/lists.txt

```diff
@@ -39,8 +39,16 @@ guidelines when introducing lists.
        period instead of a colon.
 
        .. note::
-          Avoid using fragments to introduce lists. Fragments can be
-          harder to understand than sentences.
+
+          Avoid using a fragment to introduce a list. Users might 
+          find a fragment more difficult to understand than a full 
+          sentence. If a full sentence makes the introduction awkward 
+          or redundant, you may use a stem sentence where the items in 
+          the list complete a full sentence. Consider accessibility and 
```

Comment:
Suggestion:

I don't know if there's an official term, but I was only able to find references to the term "sentence stem", and not "stem sentence". I thought the direction of only using sentence stems if the list items already fit the sentence was the opposite of what I would try, which is to update the list items to complete the sentence. Here's how I would reword these lines:

```suggestion
          or redundant, use a sentence stem to introduce the list and
          make sure the list items complete the sentence. Consider accessibility and 
```